### PR TITLE
Add Maestro Exceptions panel to grafana

### PR DESCRIPTION
**Pull request by dkurepa**
https://github.com/dotnet/arcade-services/issues/3427

### CODE REVIEW

File: src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/arcadeAvailability.dashboard.json [hunk 1]
```diff
@@ -1825,6 +1825,160 @@
       ],
       "title": "Helix AutoScaler Service Traces",
       "type": "timeseries"
+    },
+    {
+      "alert": {
+        "alertRuleTags": {},
```
Comment (dkurepa) [inline src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/arcadeAvailability.dashboard.json:1831]: Looking at other alerts, I'm missing an `NotificationId` tag. Not sure how those are generated tho, is it just a random guid?

Comment (tkapin) [reply src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/arcadeAvailability.dashboard.json:1831]: Check with @garath

Comment (premun) [reply src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/arcadeAvailability.dashboard.json:1831]: It's a random guid I manually add when I create these in the UI. Not sure that is the right approach but definitely works.

**Comment by premun** [comment]
@dkurepa there's no alert added for this panel?

Can we add the panel to the prod dashboard so that we can see how it would react to the last month's rise in exceptions?

**Comment by dkurepa** [comment]
> @dkurepa there's no alert added for this panel?
> 
> Can we add the panel to the prod dashboard so that we can see how it would react to the last month's rise in exceptions?

I redid the entire panel and force pushed to my branch. You can see the panel I created and exported here https://dotnet-eng-grafana.westus2.cloudapp.azure.com/d/arcadeAvailability/service-availability?orgId=1&from=now-30d&to=now&viewPanel=46

**Comment by premun** [comment]
Can you regenerate it again? I added a threshold so it's obvious when it will trigger

**Comment by dkurepa** [comment]
> threshold

Sure, done

**Review by premun — APPROVED** [review]

**Pull request merged**